src/controllers/transfers.js

```diff
@@ -293,7 +293,7 @@ function * putResource () {
     )
   }
 
-  transfer.id = id.toLowerCase()
+  transfer.id = uri.make('transfer', id.toLowerCase())
```

Comment:
The old version didn't cause any errors, but with https://github.com/interledgerjs/five-bells-shared/pull/169 it does because the UUID no longer matches the URI schema.